src/abstractdataframe/reshape.jl

```diff
@@ -212,7 +212,7 @@ function unstack(df::AbstractDataFrame, colkey::Int, value::Int)
         j = Int(keycol.refs[k])
         i = rowkey[k]
         if i > 0 && j > 0
-            if nowarning && !isna(df2[j][i])
+            if nowarning && !isna.(df2[j][i])
```

Comment:
Good catch. I probably shouldn't have replaced all occurrences and reverted those who triggered test failures. These lines do not appear to be covered by tests, I think Cameron added some in DataTables recently.